sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/models/AttestationSigner.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.attestation.models;
+import com.azure.core.util.logging.ClientLogger;
+
+import java.io.ByteArrayInputStream;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.util.Arrays;
+
+/**
+ * Represents an attestation signing certificate returned by the attestation service.
+ */
+public class AttestationSigner {
```

Comment:
>AttestationSigner [](http://example.com/codeflow?start=13&length=17)

Public model types should be fluent - [https://azure.github.io/azure-sdk/java_introduction.html#model-types](https://azure.github.io/azure-sdk/java_introduction.html#model-types)

Reply:
@@@#$ - made that change locally, the change didn't make this set of changes. Adding it.

Reply:
Btw, shouldn't this kind of check be in the CheckStyle plugin?

Reply:
We actually do have that. And I see you have decorated your class with @Fluent as well. I will have to check why it did not kick in.